backend/src/db/migrations/32-event-slide-data.sql

```diff
@@ -0,0 +1,31 @@
+-- Adds two new fields to `events`:
+-- `slide_text` which holds a url pointing to the extracted slide text
+-- generated in Opencast,
+-- and `segments`, which holds a list of frames with their respective
+-- starting time, and is needed for slide previews in paella.
+-- Also creates the appropriate type for the segments and adjusts
+-- the constraints. Basically an adjusted copy of `14-event-captions`.
+
+create type event_segment as (
+    uri text,
+    start_time text
```

Comment:
I think I would prefer it, if we stored this like `duration`, namely as number of milliseconds as an int. The string takes more space and I don't think we can use it for anything. The only "consumer" of this number seems to be Paella, which wants seconds anyway. Converting from milliseconds to seconds is obv much easier than this string parsing. So that basically means moving `ocTimeStringToSeconds` into the harvest code and converting before writing it to the DB. Not sure what we should do in case the parsing fails... probably just ignore all segments?

It would have been even better to already just send the milliseconds in the harvest API, but I didn't catch that when looking at the OC PR. Although... we could still fix it? It's a Tobira internal API and Tobira has not used that yet, so we can fix it without bumping the API version... mhhh